ACE/apps/gperf/src/Options.cpp

```diff
@@ -434,10 +436,10 @@ Options::parse_args (int argc, ACE_TCHAR *argv[])
                              "-v\tPrints out the current version number and exits with a value of 0\n"
                              "-V\tExits silently with a value of 0.\n"
                              "-Z\tAllow user to specify name of generated C++ class.  Default\n"
-                             "\tname is `Perfect_Hash.'\n",
+                             "\tname is `Perfect_Hash.'\n"
+                             "\t\t\tin the result, just the arguments.\n",
```

Comment:
Should this line be here?

Reply:
No, it's left over from `--record-arguments-only`. Good catch.